curriculum/challenges/english/02-javascript-algorithms-and-data-structures/debugging/catch-mixed-usage-of-single-and-double-quotes.md

```diff
@@ -16,7 +16,7 @@ Here are some examples of mixing quotes:
 
 ```js
 const grouchoContraction = "I've had a perfectly wonderful evening, but this wasn't it.";
-const quoteInString = "Groucho Marx once said 'Quote me as saying I was mis-quoted.'";
+const quoteInString = "Groucho Marx once said 'Quote me as saying I was misquoted.'";
```

Comment:
We should check the original quote

Reply:
Should be reverted.
https://www.google.com/search?q=Groucho+Marx++Quote+me+as+saying+I+was+misquoted.

Reply:
Done.